package.json

```diff
@@ -19,10 +19,10 @@
         "@tiptap/extension-link": "^2.5.0",
         "@tiptap/extension-mention": "^2.5.0",
         "@tiptap/extension-placeholder": "^2.5.0",
-        "@tiptap/extension-table": "^2.5.0",
-        "@tiptap/extension-table-cell": "^2.5.0",
-        "@tiptap/extension-table-header": "^2.5.0",
-        "@tiptap/extension-table-row": "^2.5.0",
+        "@tiptap/extension-table": "^2.8.0",
+        "@tiptap/extension-table-cell": "^2.8.0",
+        "@tiptap/extension-table-header": "^2.8.0",
+        "@tiptap/extension-table-row": "^2.8.0",
         "@tiptap/extension-task-item": "^2.5.0",
         "@tiptap/extension-task-list": "^2.5.0",
         "@tiptap/extension-text-align": "^2.5.0",
```

Comment:
Attention a bien tout mettre en 2.8.0, on évite le mix de version
Et si jamais ici exceptionnellement on a laissé les versions en dynamique et pas figé à cause d'un soucis côté tiptap

Reply:
@AlexisJC Done, fixed to 2.8.0